src/mn_setup.c

```diff
@@ -2650,35 +2674,35 @@ static setup_menu_t eq_settings1[] = {
 
     MI_GAP_EX(4),
 
-    {"Preamp dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Preamp dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_preamp"}, .action = I_OAL_EqualizerPreset},
 
     MI_GAP_EX(4),
 
-    {"Low Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Low Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_low_gain"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 1 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 1 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_mid1_gain"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 2 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 2 Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_mid2_gain"}, .action = I_OAL_EqualizerPreset},
 
-    {"High Gain dB", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"High Gain dB", S_THERMO, CNTR_X, M_THRM_SPC,
      {"snd_eq_high_gain"}, .action = I_OAL_EqualizerPreset},
 
     MI_GAP_EX(4),
 
-    {"Low Cutoff Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Low Cutoff Hz", S_NUM, CNTR_X, M_SPC,
      {"snd_eq_low_cutoff"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 1 Center Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 1 Center Hz", S_NUM, CNTR_X, M_SPC,
      {"snd_eq_mid1_center"}, .action = I_OAL_EqualizerPreset},
 
-    {"Mid 2 Center Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"Mid 2 Center Hz", S_NUM, CNTR_X, M_SPC,
      {"snd_eq_mid2_center"}, .action = I_OAL_EqualizerPreset},
 
-    {"High Cutoff Hz", S_THERMO, CNTR_X, M_THRM_SPC_EQ,
+    {"High Cutoff Hz", S_NUM, CNTR_X, M_SPC,
```

Comment:
One disadvantage in changing thermos to number entries (here and on the MIDI page) is that there's no way to know what the range limits are without guessing. Should we change the behavior so that instead of just printing a warning about being out of range, it also coerces the value to the min/max? Or would that create other problems (prefer to discard bad number entry in case of accidental entry)?

Reply:
> Should we change the behavior so that instead of just printing a warning about being out of range, it also coerces the value to the min/max?

Yes, I think it's a good idea. I will try it in separate PR.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1931/commits/a316195da0d76dbf90b764c636e68888f3e4c41b